components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/provisioning/impl/DefaultProvisioningHandler.java

```diff
@@ -212,6 +215,7 @@ tobeDeleted claims (claims came from federated idp as null). If there is a match
                     need to write a provisioning handler extending the "DefaultProvisioningHandler".
                      */
                     UserCoreUtil.setSkipPasswordPatternValidationThreadLocal(true);
+                    setSource(tenantDomain, idp, userClaims);
```

Comment:
backward compatibility should be preserved. hence use the EnhancedJITFeatures property in each places

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3583/commits/5b97c5349125240f99525e4f9e45973cf507ad9a